docs/cli/code-push/release.md

```diff
@@ -18,6 +18,10 @@ ern code-push release
 
 ### Options
 
+`--baseComposite <compositePath>`
```

Comment:
We can mention support for this command in section https://native.electrode.io/reference/index#through-commands

Reply:
Good point :+1: post merge though ... 
Will open another PR for this